pkg/kubectl/cmd/config/config.go

```diff
@@ -50,10 +64,10 @@ func NewCmdConfig(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	}
 
 	// file paths are common to all sub commands
-	cmd.PersistentFlags().BoolVar(&pathOptions.local, "local", false, "use the .kubeconfig in the current directory")
-	cmd.PersistentFlags().BoolVar(&pathOptions.global, "global", false, "use the .kubeconfig from "+os.Getenv("HOME"))
-	cmd.PersistentFlags().BoolVar(&pathOptions.envvar, "envvar", false, "use the .kubeconfig from $KUBECONFIG")
-	cmd.PersistentFlags().StringVar(&pathOptions.specifiedFile, "kubeconfig", "", "use a particular .kubeconfig file")
+	cmd.PersistentFlags().BoolVar(&pathOptions.Local, "local", pathOptions.Local, "use the .kubeconfig in the current directory")
+	cmd.PersistentFlags().BoolVar(&pathOptions.Global, "global", pathOptions.Global, "use the .kubeconfig from "+pathOptions.GlobalFile)
```

Comment:
Can we just call it `kubeconfig` ? We only the file be named `.kubeconfig` for a local one. The user can name it whatever they want for envvar or commandline, and the global one should probably not be hidden anyway (see below).